onnx/defs/operator_sets.h

```diff
@@ -146,6 +147,7 @@ class OpSet_Onnx_ver1 {
            Onnx, 1, DepthToSpace)>());
     fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 1, Div)>());
     fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 1, Dropout)>());
+    fn(GetOpSchema<ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 9, DynamicSlice)>());
```

Comment:
This should be put in OpSet_Onnx_ver9:: ForEachSchema (you need to create the new OpSet_Onnx_ver9 since this is the first change in opset 9).
Mixing schemas in different opset versions will cause issue when __ONNX_DISABLE_STATIC_REGISTRATION is false.